.gitignore

```diff
@@ -53,3 +53,4 @@ TAGS
 tags
 
 .idea
+.DS_Store
```

Comment:
I have doubts the maintainers want Mac-specific ignores to be added as part of a single board definition! Please remove.